tensorflow_serving/example/resnet_client.py

```diff
@@ -48,18 +48,18 @@ def main():
   dl_request.raise_for_status()
 
   # Compose a JSON Predict request (send JPEG image in base64).
-  predict_request = '{"instances" : [{"b64": "%s"}]}' % base64.b64encode(
+  predict_request = b'{"instances" : [{"b64": "%s"}]}' % base64.b64encode(
```

Comment:
Why is this necessary?

Reply:
Also this seems to treat the entire string as bytes rather than just the base64 encoding.

Reply:
It is necessary for python3, it does not work otherwise.

Reply:
In python3, the default is unicode which does not comply with base64.
[Here is a link to a related stack overflow issue.](https://stackoverflow.com/questions/8908287/why-do-i-need-b-to-encode-a-python-string-with-base64)

Reply:
Shouldn't just the b64 data be encoded?

```python
jpeg_bytes = dl_request.content.encode("utf-8")
predict_request = '{"instances" : [{"b64": "%s"}]}' % base64.b64encode(jpeg_bytes)
```